12-target-details.tex

```diff
@@ -301,6 +301,46 @@ \section{Neko}
 
 \section{PHP}
 \label{target-php}
+\state{NoContent}
+
+\subsection{Getting started with PHP}
+\label{target-php-getting-started}
+
+Haxe can be a powerful tool for developing PHP applications. Let's look at our first example showing the toolchain. 
+
+Create a new folder and save this class as \ic{Main.hx}.
+
+\haxe{assets/HelloPHP.hx}
+
+To compile, either run this from the command line:
+
+\begin{lstlisting}
+haxe -php bin -main Main
+\end{lstlisting}
+
+.. or create and run (double-click) a file called \ic{compile.hxml}. In this example the hxml-file should be in the same directory as the example class.
+
+\begin{lstlisting}
+-php bin
+-main Main
+\end{lstlisting}
+
+The compiler outputs in the given \emph{bin}-folder, which contains the generated PHP classes that prints the traced message when you run it.
+
+\paragraph{Run the PHP}
+
+To run the PHP a web server is needed, such as \href{http://httpd.apache.org/}{Apache}. An easy-to-install Apache distribution for local development is \href{https://www.apachefriends.org/download.html}{XAMPP}.
```

Comment:
That's overkill, all you need is a PHP interpreter. You can easily run PHP from command line (I do it all the time when testing).